ts/components/session/LeftPaneMessageSection.tsx

```diff
@@ -97,7 +97,7 @@ export class LeftPaneMessageSection extends React.Component<Props, any> {
     if (conversationList !== undefined) {
       conversationList = conversationList.filter(
         conversation =>
-          !conversation.isSecondary && !conversation.isPendingFriendRequest
+          !conversation.isPendingFriendRequest && !conversation.isSecondary
```

Comment:
based on the state of the redux store you made, conversation.isSecondary should never happen. but we can keep it for clarity